plugins/module_utils/prism/service_groups.py

```diff
@@ -22,6 +29,70 @@ def get_uuid(self, value, key="name", raise_error=True, no_response=False):
                     return entity["uuid"]
         return None
 
+    def _get_default_spec(self):
+        return deepcopy(
+            {
+                "name": None,
+                "service_list": [],
+            }
+        )
+
+    def _build_spec_name(self, payload, value):
+        payload["name"] = value
+        return payload, None
+
+    def _build_spec_desc(self, payload, value):
+        payload["description"] = value
+        return payload, None
+
+    def _build_spec_services(self, payload, config):
+        service_list = []
+
+        if config.get("tcp"):
+            service = {}
+            service["protocol"] = "TCP"
+            port_range_list = []
+            if "*" not in config["tcp"]:
+                for port in config["tcp"]:
+                    port = port.split("-")
+                    port_range_list.append(
+                        {"start_port": int(port[0]), "end_port": int(port[-1])}
```

Comment:
Here 80 or 80-90 or 80,90,100 or Any are allowed. Can we put some validation to not allow something else here ?
Also instead of using "*" Can we use "Any" which is also used in UI for referring all ports.

Reply:
Decide if "Any" or "*" to be used. Also maintain same for icmp code